src/richie/apps/courses/templates/courses/cms/fragment_course_content.html

```diff
@@ -68,15 +68,13 @@ <h2 class="course-detail__content__row__title">{% trans 'Assessment and certific
   <h2 class="section__title course-detail__content__row__title course-detail__content__organizations__title">
     {% trans 'Organizations' %}
   </h2>
-  {% if page|is_empty_placeholder:"course_organizations" %}
-    <p class="course-detail__content__organizations__placeholder">
-      {% trans "What are the organizations publishing this course?" %}
-    </p>
-  {% else %}
-    <div class="section__items course-detail__content__organizations__items">
-      {% page_placeholder "course_organizations" page %}
-    </div>
-  {% endif %}
+  <div class="section__items course-detail__content__organizations__items">
```

Comment:
Maybe wrap it in a test (like it's done here https://github.com/openfun/richie/blob/master/src/richie/apps/courses/templates/courses/cms/course_detail.html#L38) so we don't get an empty div on the production site:
```django
{% if current_page.publisher_is_draft or not current_page|is_empty_placeholder:"course_organizations" %}
```

Reply:
The condition is on line 66:
https://github.com/openfun/richie/blob/5bd8614323e5ea791f42037f800e1380baa5e9e5/src/richie/apps/courses/templates/courses/cms/fragment_course_content.html#L66
The condition I'm removing on line 71 was preventing the placeholder from appearing even on draft page if it wasn't already populated.

Reply:
OK ! Sorry for the hassle.

Reply:
No worries ! 👍 Thanks for the reviews !